nucliadb_utils/nucliadb_utils/storages/gcs.py

```diff
@@ -683,7 +683,8 @@ async def delete_kb(self, kbid: str):
                 logger.info(f"Deleted bucket: {bucket_name}")
                 deleted = True
             if resp.status == 409:
-                logger.info(f"Conflict on deleting: {bucket_name}")
+                details = resp.text()
+                logger.info(f"Conflict on deleting bucket {bucket_name}: {details}")
```

Comment:
can you add a fallback for status codes that are neither 200, 409 or 404? right now we would be ignoring them